model_compression_toolkit/gptq/pytorch/graph_info.py

```diff
@@ -12,24 +12,25 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 # ==============================================================================
-import torch
 import torch.nn as nn
 from typing import List
-from model_compression_toolkit.gptq.pytorch.quantizer.quantizer_wrapper import WeightQuantizerWrapper
+
 from model_compression_toolkit.core.pytorch.constants import BIAS
+from model_compression_toolkit.core.pytorch.default_framework_info import DEFAULT_PYTORCH_INFO
+from model_compression_toolkit.gptq.common.gptq_graph import get_kernel_attribute_name_for_gptq
+from model_compression_toolkit.quantizers_infrastructure import PytorchQuantizationWrapper
 
 
-def get_trainable_parameters(fxp_model: nn.Module,
-                             add_bias: bool = False,
-                             quantization_parameters_learning: bool = False
-                             ) -> (List[nn.Parameter], List[nn.Parameter], List[nn.Parameter]):
+def gptq_get_trainable_parameters(fxp_model: nn.Module,
```

Comment:
I wanted to make it similar to the matching Keras function.
In general, I tried to make the "signature" of matching components similar as possible between frameworks, to prevent confusion.